test/memcache-helper.js

```diff
@@ -19,7 +19,14 @@ var config = {
     maxBadLogins: 2,
     maxBadLoginsPerIp: Number(process.env.MAX_BAD_LOGINS_PER_IP) || 3,
     ipRateLimitIntervalSeconds: Number(process.env.IP_RATE_LIMIT_INTERVAL_SECONDS) || 60 * 15,
-    ipRateLimitBanDurationSeconds: Number(process.env.IP_RATE_LIMIT_BAN_DURATION_SECONDS) || 60 * 15
+    ipRateLimitBanDurationSeconds: Number(process.env.IP_RATE_LIMIT_BAN_DURATION_SECONDS) || 60 * 15,
+    badLoginLockout: 3,
+    badLoginLockoutIntervalSeconds: 20,
```

Comment:
These lockout-related settings have been removed in master, I wonder if they came back in through a rebase or something.  Anyway, they're not needed, please remove.